packages/styles/stepper.css

```diff
@@ -1,12 +1,58 @@
-.Stepper {
-  --step-indicator-size: 2em;
+:root {
+  --step-indicator-size: 1.867em;
   --step-line-height: 0.53em;
-  --step-line-offset: 1.33em;
+  --step-line-offset: 1.53em;
```

Comment:
I realize these were `em` before, but they should probably be `rem` so they don't scale relative to the inherited font size but the root font size.

I think `step-indicator-size` needs to be `1.75rem` to match the `28px` set in the uxpin. The others should be updated to match accordingly.

And FYI, line-height values are already relative, so `em` can be omitted entirely, instead specifying the number without the unit.